pkg/sql/execinfrapb/processors_export.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2025 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+//
+// Processor definitions for distributed SQL APIs. See
+// docs/RFCS/distributed_sql.md.
+// All the concepts here are "physical plan" concepts.
+
+syntax = "proto2";
```

Comment:
nit: I'd prefer to see this code move (and CODEOWNERS file change) in a separate commit